source/Globals.h

```diff
@@ -22,15 +19,36 @@ constexpr unsigned int NETWORK_TICK = 12; // NOLINT
 /** Health of the ship to start */
 constexpr int INITIAL_SHIP_HEALTH = 11; // NOLINT
 
-/** Radius of the ship for the scene graph */
-constexpr unsigned int RADIUS = 550; // NOLINT
+/** Minimum of ship health for win condition */
+constexpr int SHIP_HEALTH_WIN_LIMIT = 10;
```

Comment:
You need to add NOLINT comments to your constants so the linter doesn't show warnings

Reply:
Only in `Globals.h`. I think there's a slight problem with how I've set up this class; I'll look into it in a future sprint. For now, yes, please add `// NOLINT` after each constant inside `Globals.h`.

Reply:
Done!